h2o-core/src/main/java/water/jdbc/SQLManager.java

```diff
@@ -20,10 +20,12 @@
   //A lower bound on number of connections to database per node
   private static final int MIN_CONNECTIONS_PER_NODE = 1;
   private static final String NETEZZA_DB_TYPE = "netezza";
+  private static final String HIVE_DB_TYPE = "hive2";
```

Comment:
How do you want to support hive1 ?

Reply:
It is harder - because Hive1 doesn't support OFFSET specification - that makes it harder to do the distributed ingest. My suggestion is to stay with just Hive2 for now and add Hive1 in the future - new jira.